TYPO3.Flow/Classes/TYPO3/Flow/Reflection/ObjectAccess.php

```diff
@@ -394,6 +419,16 @@ public static function isPropertyGettable($object, $propertyName)
     }
 
     /**
+     * Check if the given $value is any type that can be assigned to a collection type property.
+     *
+     * @return boolean TRUE if $value is either NULL, an array or an instance of \Traversable
+     */
+    public static function isCollectionTypePropertyValue($value)
```

Comment:
I think we should use the existing `TypeHandling::isCollectionType()` method here!